ffs/manager/manager.go

```diff
@@ -85,8 +88,40 @@ func (m *Manager) Create(ctx context.Context) (ffs.APIID, string, error) {
 	defer m.lock.Unlock()
 
 	log.Info("creating instance")
+
+	var addr address.Address
+	if m.ffsUseMasterAddr {
+		addr = m.wm.MasterAddr()
```

Comment:
Important: We should cover here that the WalletManager could not have a master address.
As a refresher, there're three possibilities:
- The masteraddr flag nor autocreatemaster are set, in that case there's no masteraddr. That means that newly created FFS instance won't be autofunded. (Since there isn't any addr as the source of the transaction).
- In any other case, there's a masteraddr. Being directly set with the masteraddr flag, or with the autocreatemaster.

So the new flag of this PR should check that if `true`, then at least one of both flags are set. 
Sounds that should happen in `server.go` as a validation while bootstrapping things. 

I'd also add some logic here to also see if `.MasterAddr()` returns the default value (or maybe change signature to also return a bool to indicate if exist). As mentioned in the above comment, that API should have a clear way of saying: _There isn't any configured masteraddr._

Reply:
👍

Reply:
Decided to move the check to the `manager.New()` so that if there's a problem the error happens on start up rather than waiting for someone to create an FFS instance. This still is really just a backup because there is also not a check in `server.go` that should catch this problem even sooner.